src/schema/v2/order/sharedTypes.ts

```diff
@@ -0,0 +1,449 @@
+
+        return resolveMinorAndCurrencyFieldsToMoney(
+          { minor, currencyCode },
+          _args,
+          context,
+          _info
+        )
+      },
+    },
+    selected: { type: GraphQLBoolean },
+  },
+})
+
+const FulfillmentDetailsType = new GraphQLObjectType<any, ResolverContext>({
+  name: "FulfillmentDetails",
+  description: "Buyer fulfillment details for order",
+  fields: {
+    phoneNumber: {
+      type: GraphQLString,
+      description: "Phone number of the buyer",
+    },
+    phoneNumberCountryCode: {
+      type: GraphQLString,
+      description: "Country code of the buyer's phone number",
+    },
+    name: {
+      type: GraphQLString,
+      description: "Name line for shipping address",
+    },
+    addressLine1: {
+      type: GraphQLString,
+      description: "Shipping address line 1",
+    },
+    addressLine2: {
+      type: GraphQLString,
+      description: "Shipping address line 2",
+    },
+    city: {
+      type: GraphQLString,
+      description: "Shipping address city",
+    },
+    postalCode: {
+      type: GraphQLString,
+      description: "Shipping address postal code",
+    },
+    region: {
+      type: GraphQLString,
+      description: "Shipping address state/province/region",
+    },
+    country: {
+      type: GraphQLString,
+      description: "Shipping address country",
+    },
+  },
+})
+
+const LineItemType = new GraphQLObjectType<any, ResolverContext>({
+  name: "LineItem",
+  description: "A line item in an order",
+  fields: {
+    ...InternalIDFields,
+    artwork: {
+      type: ArtworkType,
+      resolve: ({ artwork_id }, _args, { artworkLoader }) => {
+        return artworkLoader(artwork_id)
+      },
+    },
+    artworkVersion: {
+      type: ArtworkVersionType,
+      resolve: (
+        { artwork_version_id },
+        _args,
+        { authenticatedArtworkVersionLoader }
+      ) =>
+        authenticatedArtworkVersionLoader
+          ? authenticatedArtworkVersionLoader(artwork_version_id)
+          : null,
+    },
+    currencyCode: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ currency_code }) => currency_code,
+    },
+    listPrice: {
+      type: Money,
+      resolve: async (
+        // TODO: Remove USD fallback and include currency_code in the line item json
+        { list_price_cents: minor, currency_code: currencyCode = "USD" },
+        _args,
+        context,
+        _info
+      ) => {
+        return resolveMinorAndCurrencyFieldsToMoney(
+          {
+            minor: minor,
+            currencyCode,
+          },
+          _args,
+          context,
+          _info
+        )
+      },
+    },
+    quantity: {
+      type: GraphQLNonNull(GraphQLInt),
+      resolve: ({ quantity }) => quantity,
+    },
+  },
+})
+
+export const OrderType = new GraphQLObjectType<OrderJSON, ResolverContext>({
+  name: "Order",
+  description: "Buyer's representation of an order",
+  fields: {
```

Comment:
This single file now contains the shared types of The Order

Reply:
Can we rename the file to `sharedOrderTypes`? so that its a quick (by memory) cmd+p away?